lmdeploy/model.py

```diff
@@ -1472,6 +1472,8 @@ def best_match_model(query: str) -> Optional[str]:
         from transformers import AutoTokenizer
         tokenizer_config = AutoTokenizer.from_pretrained(
             query, trust_remote_code=True)
+        if getattr('tokenizer_config', 'chat_template', None) is None:
+            return 'base'
```

Comment:
Should we remove the below condition `if tokenizer_config.chat_template`?

Reply:
The try .. except seems useless. Can we just return 'base' if query not in MODELS.module_dict.items() ?